src/components/organisms/NavBarSchedule/NavBarSchedule.scss

```
@@ -1,69 +1,39 @@
 @import "scss/constants.scss";
 
-$NavBarSchedule--width: 40%;
-$NavBarSchedule--padding: 20px;
+$navbar-schedule-width: 40%;
+$navbar-schedule-padding: 20px;
 
-$weekday--padding: 8px 10px;
-$weekday--margin-right: 10px;
+$weekday-padding: 8px 10px;
+$weekday-margin-right: 10px;
 
-$schedule-btn--padding: 0.35em 1em;
+$schedule-btn-padding: 0.35em 1em;
```

No harm in full word `button`

```suggestion
$schedule-button-padding: 0.35em 1em;
```

If accepted, rename like that on places of usage as well.